File: python/fizzbuzz.py
FIZZ = 'Fizz'
BUZZ = 'Buzz'
STOP = 100

for i in range(STOP):
	i += 1
	if i % 15 == 0:
		print(FIZZ + BUZZ)
	elif i % 3 == 0:
		print(FIZZ)
	elif i % 5 == 0:
		print(BUZZ)
	else:
		print(i)

File: javascript/fizzbuzz.js
const FIZZ = 'Fizz';
const BUZZ = 'Buzz';
const STOP = 100;

for (let i = 1; i <= STOP; i++) {
    if (i % 15 === 0) {
        console.log(FIZZ + BUZZ);
    } else if (i % 3 === 0) {
        console.log(FIZZ);
    } else if (i % 5 === 0) {
        console.log(BUZZ);
    } else {
        console.log(i);
    }
}
